package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@synergy-design-system/root",
   "version": "0.1.0",
-  "homepage": "https://github.com/synergy-design-system/synergy",
+  "homepage": "https://synergy-design-system.github.io",
```

Comment:
As said: THis change is especially important in the packages, as only they are provided on NPM.